src/backend/helpers/pluginHelpers.ts

```diff
@@ -42,7 +42,7 @@ export const deleteInstalledPlugin = (
 export const updatePluginSettings = (
   supabase: SupabaseClient,
   project_id: string,
-  plugin_id: string,
+  plugin_name: string,
```

Comment:
I just added this while I was at it. This fixes a problem that plugin settings could not be updated from the "Settings" page.